lib/index.js

```diff
@@ -192,13 +200,11 @@ class Configurator {
 		// TODO: these modules need to be transpiled
 		// NOTE: these are npm modules, babel has a freak out if you just include all of /node_modules/
 		const es6Modules = [
-			'@financial-times/n-card',
-			'@financial-times/n-image',
-			'@financial-times/n-section',
-			'@financial-times/n-ui'
+			'@financial-times/n-'
```

Comment:
I mean n- modules that are already babelified will still be parsed, etc unnecessarily by babelify.